mock/moira-alert/logger.go

```diff
@@ -116,26 +126,30 @@ func (m *MockLogger) Fatalf(arg0 string, arg1 ...interface{}) {
 
 // Fatalf indicates an expected call of Fatalf
 func (mr *MockLoggerMockRecorder) Fatalf(arg0 interface{}, arg1 ...interface{}) *gomock.Call {
+	mr.mock.ctrl.T.Helper()
 	varargs := append([]interface{}{arg0}, arg1...)
 	return mr.mock.ctrl.RecordCallWithMethodType(mr.mock, "Fatalf", reflect.TypeOf((*MockLogger)(nil).Fatalf), varargs...)
 }
 
-// Maintenance mocks base method
+// Info mocks base method
```

Comment:
Why this methods was renamed?

Reply:
It seems, that the info method mock was rewritten by hand to call mock by name "Maintenance" instead of "Info". I don't see any reason for why that was done. Commit 614867417ab. 
AFAIK this has no effect on the mock's behaviour. Regenerating the mock code produced a correct version.